components/cap-case.js

```diff
@@ -0,0 +1,52 @@
+import { LitElement, html, css, unsafeHTML } from "../lib/lit.js";
+
+import { fetchCaselawBody } from "../lib/data.js";
+
+export default class CapCase extends LitElement {
+	static properties = {
+		caseBody: { attribute: false },
+		reporter: { type: String },
+		volume: { type: String },
+		case: { type: String },
+	};
+
+	constructor() {
+		super();
+		this.caseBody = "";
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		fetchCaselawBody(
+			this.reporter,
+			this.volume,
+			this.case,
+			(data) => (this.caseBody = data)
+		);
+	}
```

Comment:
This definitely feels like it's easier to read, and feels more closely aligned to how, in a component-driven paradigm, a function might update the component's internal state using a callback rather than directly modifying the state itself